src/components/PackageList/PackageList.tsx

```diff
@@ -12,22 +12,23 @@ interface Props {
   packages: PackageInterface[];
 }
 
-export const PackageList: React.FC<Props> = props => {
-  const renderPackages: () => React.ReactElement<HTMLElement>[] = () => {
-    return props.packages.map((pkg, i) => {
-      const { name, version, description, time, keywords, dist, homepage, bugs, author } = pkg;
+export const PackageList: React.FC<Props> = ({ packages }) => {
+  const renderPackages: () => ReactNode[] = () => {
+    return packages.map(({ name, version, description, time, keywords, dist, homepage, bugs, author, license }, i) => {
       // TODO: move format license to API side.
-      const license = formatLicense(pkg.license);
+      const _license = formatLicense(license);
```

Comment:
Please why did you add `_` ?

Reply:
we cant name this variable `license` the props already have a prop named `license` I presume this why the component was not destructuring the props before. this way we could call `pkg.license`